README.md

```diff
@@ -18,7 +18,45 @@ To scrape a git repository for probe definitions, an entry needs to be added in
 - `notification_emails`: Where emails about probe-scraper failures and improper files will be forwarded to. These
 will be just about your specific repository.
 - `url`: The URL of the repository to scrape. It should be able to be cloned directly from that URL.
-- `metrics_files`: A list of relative paths to `metrics.yaml` files
+- `metrics_files**: A list of relative paths to `metrics.yaml** files
+
+### Adding an application
```

Comment:
Should this be mentioned in the Glean SDK docs as well? Or at least reference this document. Or, better, should we have "one true doc" somewhere about "Using Glean" (SDK, Pipeline and whatnot). Not for this PR, but we need to track this somewhere I think.

Reply:
We have [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1529761) about unifying Glean documentation.  I added a note that we have this documentation here as well that should be pulled in.